tests/test_simultaneous_initiate.c

```diff
@@ -1207,6 +1262,17 @@ START_TEST(test_repeated_simultaneous_initiate_lost_message_repeated_messages)
         result = session_builder_process_pre_key_bundle(bob_session_builder, alice_pre_key_bundle);
```

Comment:
Same thing here and even in this line https://github.com/nealrea/libsignal-protocol-c/blob/3aebf8c0749146fd845c4d46a2992bfe3cade7fd/tests/test_simultaneous_initiate.c#L1239